src/operations.js

```diff
@@ -28,10 +28,19 @@ module.exports = function (model, options, excludedMap) {
 
     options.contextFilter(model, req, (filteredContext) => {
       buildQuery(filteredContext.find(), req._ermQueryOptions).then((items) => {
+        var totalCountHeader;
         req.erm.result = items
         req.erm.statusCode = 200
 
-        if (options.totalCountHeader && !req._ermQueryOptions['distinct']) {
+        if (req.query.totalCountHeader === 'true') {
+          totalCountHeader = true;
+        } else if (req.query.totalCountHeader === 'false') {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/florianholzapfel/express-restify-mongoose/pullRequest?prid=8423229)
```suggestion
        } else if (req.query.totalCountHeader === "false") {
```